### PR TITLE
Add k8s objects

### PR DESCRIPTION
**Pull request by suprugafad**

### CODE REVIEW

**Comment by suprugafad** [comment]
Closes #4

File: src/kubernetes/pg-statefulset.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    matchLabels:
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres:latest
```
Comment (h1alexbel) [inline src/kubernetes/pg-statefulset.yaml]: @suprugafad the same here

File: src/kubernetes/nestjs-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              valueFrom:
+                secretKeyRef:
+                  name: pg-secrets
+                  key: POSTGRES_PASSWORD
+          envFrom:
+            - configMapRef:
+                name: nestjs-configmap
+      containers:
+        - name: nestjs
+          image: suprugafad/medium-clone:latest
```
Comment (h1alexbel) [inline src/kubernetes/nestjs-deployment.yaml]: @suprugafad latest image version is not a good idea

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/kubernetes/nestjs-deployment.yaml (hunk 2 above)
Comment (suprugafad) [reply src/kubernetes/nestjs-deployment.yaml]: fixed

File: src/kubernetes/pg-statefulset.yaml (hunk 1 above)
Comment (suprugafad) [reply src/kubernetes/pg-statefulset.yaml]: @h1alexbel fixed

File: src/kubernetes/nestjs-deployment.yaml [hunk 3]
```diff
@@ -32,7 +32,7 @@ spec:
                 name: nestjs-configmap
       containers:
         - name: nestjs
-          image: suprugafad/medium-clone:latest
+          image: suprugafad/medium-clone
```
Comment (h1alexbel) [inline src/kubernetes/nestjs-deployment.yaml]: @suprugafad what is the specific version of this image?

Comment (suprugafad) [reply src/kubernetes/nestjs-deployment.yaml]: @h1alexbel I have only one version, so now it's without tag, in future it can be with :2, i need to add it?

Comment (suprugafad) [reply src/kubernetes/nestjs-deployment.yaml]: @h1alexbel oh, i understood the problem)
without tag it will take always latest image?

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**